src/web/views/enter-code.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "templates/form.njk" %}
+{% from "input/macro.njk" import govukInput %}
+{% from "fieldset/macro.njk" import govukFieldset %}
+{% from "details/macro.njk" import govukDetails %}
+
+{% block formContent %}
+
+    {% call govukFieldset({
+        legend: {
+            text: heading,
+            classes: "govuk-fieldset__legend--xl",
+            isPageHeading: true
+        }
+    }) %}
+
+        <p class="govuk-body-m">{{ weJustSentASixDigitCodeTo}} <strong>{{ emailOrPhoneNumberForConfirmationCode }}</strong>. {{ itMayTakeAFewMinutesToArrive}}</p>
```

Comment:
I think this would this be better as a single translated bit of text like this one:
```
"You are entitled to\n£{{totalVoucherValue}} per week"
```

Reply:
Needs to be broken up because of the `strong` element.